### ISSUE TITLE
anti-nucleosome antibody

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11634]
**Preferred term label:**
anti-nucleosome antibody

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
addede
